### PR TITLE
Suggestion Update HeadOfProfession.md

### PR DESCRIPTION
**Pull request by JaskiranKang**
A couple of extra bullets added to this. Please review.

### CODE REVIEW

File: Head of Profession/HeadOfProfession.md [hunk 1]
```diff
@@ -20,6 +21,8 @@
 ## Quality and standards
 - Creating accessible knowledge, standards, and practice to set good conditions for people in their professions to work effectively, and ensuring they are well enforced through service assessments
 - Being the guardian of quality and standards of the practitioners within the profession and the principles and policies of the profession
+- Lead by example
+- Grow assessors in the profession to assurance in-house/in-delivery services. (Self-sustaining professions)
```
Comment (lwlsns) [inline Head of Profession/HeadOfProfession.md:25]: assure
in-delivery?

File: Head of Profession/HeadOfProfession.md [hunk 2]
```diff
@@ -7,6 +7,7 @@
 - Working across government and the sector to ensure a cohesive approach to the specialism, create opportunities for collaboration through existing networks – or setting up new networks where they don’t exist
 - Hold authority to ensure that specialists are able to contribute towards building the profession and participate in community, backed up by DDaT Committee.\*
 - Raising the profile of the profession across the Department
+- Grow the capability through attraction strategy and rentention strategy
```
Comment (lwlsns) [inline Head of Profession/HeadOfProfession.md:10]: attraction and retention strategy

**Review by lwlsns — COMMENTED** [review]
couple of small changes

**Comment by luke-slowen** [comment]
@lwlsns @JaskiranKang I've made some further tweaks to pull together your updates & comments, happy to tick this off?

**Pull request merged**